clients/java/signalr/src/main/java/HubConnection.java

```diff
@@ -55,13 +55,39 @@ public void stop(){
         connected = false;
     }
 
-    public void send(String method, Object... args) {
+    public void send(String method, Object... args) throws Exception {
         InvocationMessage invocationMessage = new InvocationMessage(method, args);
         String message = protocol.writeMessage(invocationMessage);
         transport.send(message);
     }
 
     public void On(String target, Action callback) {
```

Comment:
I think I made this comment on the earlier PR. This should be `camelCased` like the other methods (so `on` instead of `On`) 😸